obspy/io/sh/core.py

```diff
@@ -404,7 +404,7 @@ def _read_q(filename, headonly=False, data_directory=None, byteorder='=',
             id = cid
             i += 1
         traces.setdefault(i, '')
-        traces[i] += line[3:].strip()
+        traces[i] += line[3:].rstrip('\n')
```

Comment:
Ah I see. Just using `rstrip("\r\n")` should also do the trick. But the `splitlines()` would of course also be very clean.